birdhouse/config/canarie-api/entrypoint

```diff
@@ -5,5 +5,12 @@
 # remove if cause problem with newer version of canarieapi
 touch /etc/cron.d/canarieapi-cron
 
+pip install requests-cache
```

Comment:
It could, but it is not used anywhere in it.

Reply:
Or have a Pavics specific Canarie-API image?  Any other suggestions?  It just feels so wrong to install in the entrypoint like this.  This is usually a trick for short term work-around only.

Reply:
Seems overkill to generate a separate image just for that.

Reply:
Not overkill at all.  Reproductibility is the reason docker image was invented.  It gives a snapshot of all the binaires required to run a product.

We have a custom JupyterHub image simply to have the magpie_authenticator and the dockerspawner pre-installed, see https://github.com/Ouranosinc/jupyterhub/blob/master/Dockerfile

My custom Geoserver build is to have the 2 missing plugin pre-installed into the image instead of downloading them on the fly in the entrypoint, see https://github.com/tlvu/docker-geoserver/blob/2e5dbb99effa75abe818cdf551532f2b1de7739c/Dockerfile.custom

Reply:
https://github.com/bird-house/birdhouse-deploy/issues/322